xwiki-platform-distribution/xwiki-platform-distribution-flavor/xwiki-platform-distribution-flavor-test/xwiki-platform-distribution-flavor-test-ui/src/test/it/org/xwiki/test/ui/CompareVersionsTest.java

```diff
@@ -276,6 +279,94 @@ public void testUnifiedDiffShowsInlineChanges()
             "+alice = <ins>2 * </ins>bob;", " bob = tmp;");
     }
 
+    /**
+     * Tests navigation between versions.
+     */
+    private void testVersionNavigation()
+    {
+        String queryString = "viewer=changes&rev1=1.2&rev2=6.4";
+        getUtil().gotoPage(getTestClassName(), testPage.getMetaDataValue("page"), "view", queryString);
+        ChangesPane changesPane = new ChangesPane();
+        assertEquals(changesPane.getFromVersion(), "1.2");
+        assertEquals(changesPane.getToVersion(), "6.4");
+        assertTrue(changesPane.hasPreviousChange());
+        assertTrue(changesPane.hasNextChange());
+        assertTrue(changesPane.hasPreviousFromVersion());
+        assertTrue(changesPane.hasNextFromVersion());
+        assertTrue(changesPane.hasPreviousToVersion());
+        assertTrue(changesPane.hasNextToVersion());
+
+        changesPane.clickPreviousChange();
+        changesPane = new ChangesPane();
+        assertEquals(changesPane.getFromVersion(), "1.1");
+        assertEquals(changesPane.getToVersion(), "1.2");
+        assertTrue(!changesPane.hasPreviousChange());
```

Comment:
As a general note, it's clearer and recommended to use the dedicated `assertFalse`.